content-security-policy/reporting/support/preload-csp-report.https.sub.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<body>
+<!-- This image will cause a CSP violation, which will trigger an immediate report -->
+<script>
+    const href = "/reporting/resources/fail.png";
+    const uid = new URLSearchParams(location.search).get('uid');
+//    const bc = new BroadcastChannel(uid);
```

Comment:
This seems sketchy. Line that's commented out below as well.

Reply:
Oops, was not prepared to land this yet - wanted to see that we agree on the main PR. Will prepare this one.

Reply:
Done